docs/connect.html

```diff
@@ -114,8 +114,74 @@
         <li><code>transforms.$alias.$transformationSpecificConfig</code> Configuration properties for the transformation</li>
     </ul>
 
+    For example, lets take the built-in file source connector and use a transformation to add a static field. Since the file source connector reads each line as a String, we will use two transformations:
+    <ul>
+        <li><b>HoistField</b> to place the String inside a map, so we can add the static field to that map. Otherwise there's nothing to add the field to.</li>
+        <li><b>InsertFiled</b> to add the static field. In this example we'll indicate that the record came from a file connector</li>
+    </ul>
+
+    Note that "InsertSource" and "MakeMap" are the alias that I chose to give the transformations. You can give a transformation any name you'd like.
+
+    After adding transformations, the <code>connect-file-source.properties</code> file looks as following:
+
+    <pre>
+        name=local-file-source
+        connector.class=FileStreamSource
+        tasks.max=1
+        file=test.txt
+        topic=connect-test
+        transforms=MakeMap, InsertSource
+        transforms.MakeMap.type=org.apache.kafka.connect.transforms.HoistField$Value
+        transforms.MakeMap.field=line
+        transforms.InsertSource.type=org.apache.kafka.connect.transforms.InsertField$Value
+        transforms.InsertSource.static.field=data_source
+        transforms.InsertSource.static.value="test-file-source"
+    </pre>
+
+    <p>All the lines starting with <code>transforms</code> were added for the transformations. You can see the two transformations I created. The types are based on the list of built-in transformations you can see below. Each transformation type has additional configuration: HoistField requires a configuration called "field", which is the name of the field in the map that will include the original String from the file. InsertField transformation allowed me to specify the field name and the value that I am adding.</p>
+
+    <p>Since the expected output is no longer a String but rather a JSON map with two fields, I also modified the framework configuration to make the output schemaless. This is easier to read and I don't plan to use the schema anyway. So I changed the following two lines in <code>connect-standalone.properties</code>:</p>
+
+    <pre>
+        key.converter.schemas.enable
+        value.converter.schemas.enable
+    </pre>
+
+    from true to false. When I ran the file source connector on my sample file before adding transformations, and then read them using <code>kafka-console-consumer.sh</code>, the results were:
+
+    <pre>
+        "foo"
+        "bar"
+        "hello world"
+   </pre>
+
+    After adding the transformations (and deleting the existing offsets from /tmp/connect.offsets, so I can re-run the connector from the beginning), I got:
+
+    <pre>
+        {"line":"foo","data_source":"\"test-file-source\""}
```

Comment:
This output with escaped quotes in it seems to confirm my above question about extra quotes in the field value.